build.gradle

```diff
@@ -368,3 +368,32 @@ task getProjectVersion {
     }
 }
 
+import org.apache.commons.configuration.PropertiesConfiguration
+
+task incrementProjectVersion {
+    doLast{
+        PropertiesConfiguration props = new PropertiesConfiguration('gradle.properties')
+        String version = props.getProperty("version")
+        println "old version: " + version
+        String newVersion = version.tokenize(".")[0].toString() + "." + version.tokenize(".")[1].toString() + "." + (version.tokenize(".")[2].toInteger() + 1).toString()
```

Comment:
Yoy do not need to tokenize version several times.
you can declare another array variable:
```
String [] vTokens = version.tokenize(".")
String newVersion = "${vTokens[0]}.${vTokens[1]}.${vTokens[2].toInteger() + 1}".toString()
```